lib/Test/Pod/CoverageChange.pm

```diff
@@ -0,0 +1,246 @@
+use strict;
+use warnings;
+
+package Test::Pod::CoverageChange;
+# ABSTRACT: check perl modules against their pod coverage
+
+our $VERSION = '0.001';
+# AUTHORITY
+
+use utf8;
+
+=encoding utf8
+
+=head1 NAME
+
+Pod coverage calculator test file.
+
+=head1 SYNOPSIS
+
+    use Test::Pod::CoverageChange qw(pod_coverage_syntax_ok);
+
+    Test::Pod::CoverageChange::pod_coverage_syntax_ok('lib', {
+        Module::With::3::expected::naked::subs              => 3,
+        AnotherModule::With::10::expected::naked::subs      => 10,
+        YetAnotherModule::With::1::expected::naked::subs    => 1,
+        YetAnotherModule::With::5::expected::naked::subs    => 5,
+    }, [
+        We::Ignore::ThisModule,
+        We::Also::Ignore::This::Module
+    ]);
+
+=head1 DESCRIPTION
+
+It checks all files that placed under a given directory against their
```

Comment:
```suggestion
C<Test::Pod::CoverageChange> checks all files that placed under a given directory against their
```

Though this should really be consistent what what you say in `SYNOPSIS`, so e.g. with my previous suggestion, it could be something like:

```perl
=head1 DESCRIPTION

C<Test::Pod::CoverageChange> is a helper combining L<Test::Pod::Coverage> and
L<Pod::Checker> to test for both POD coverage and syntax changes for a module
distribution at once, via a single call to L</pod_coverage_syntax_ok>.

=cut
```